src/DocumentFormat.OpenXml/ParaId/IParagraphIdService.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace DocumentFormat.OpenXml.Wordprocessing
+{
+    /// <summary>
+    /// Defines the operations of the w14:paraId (ParagraphId) service.
+    /// </summary>
+    public interface IParagraphIdService
+    {
+        /// <summary>
+        /// Gets the w14:paraId values that have been registered with the
+        /// <see cref="IParagraphIdService" /> instance.
+        /// </summary>
+#if NET35 || NET40
```

Comment:
I had thought about a common API. Wanting to use an `IReadOnlyCollection<string>`, I realized that only `net35` and `net40` do not support that interface, and the corresponding interface in the SDK is declared internal (why?).

All frameworks released since 2012 (net45 and later) support the interface. Therefore, I went with `IReadOnlyCollection<string>` for anybody not stuck with `net35`. Those currently stuck with `net35` will anyhow be stuck with that framework forever (and it seems it's supported until 2029, which could be deemed a form of cruelty ;-). .NET Framework 4.0 is no longer supported since 2016, so I don't see why I should bother.